src/content/docs/client-apis/cli.mdx

```diff
@@ -121,17 +134,51 @@ Time: 0.09ms (compiling), 11.75ms (executing)
 
 ## Query progress
 
-For queries that might take a significant amount of time to execute, a progress bar can be enabled that displays
+For queries that might take a significant amount of time to execute, there is a progress bar that displays
 the number of pipelines that have been executed (each query is broken down into one or more pipelines), 
 as well as the percentage of the data processed in a pipeline. This gives an estimate for how much of a pipeline
 has executed.
 
 ![](/img/progress-bar.gif)
 
-The progress bar is not enabled by default. To enable the progress bar, use the following command:
+The progress bar is enabled by default. To disable the progress bar, use the `--no_progress_bar` flag or the following command:
```

Comment:
Sorry, forgot to comment here, we should revert this sentence too.

Reply:
Just done :)